lzma/Makefile.am

```diff
@@ -1,4 +1,4 @@
-SUBDIRS = C ASM/x86
```

Comment:
`ASM/x86` have been handled in C submodule.

Reply:
IDK who added this. But if you do this, you'll have to make sure there is nothing in the x86 dir that IS needed.

Reply:
I just wanna successfully build `lrzip 0.641`. I don't wanna change too much.